internal/logger/stderr.go

```diff
@@ -15,7 +14,7 @@ func NewDefaultLogger(service string) zerolog.Logger {
 }
 
 func NewStdErr(cf *shared.LoggerConfigFile, service string) zerolog.Logger {
-	zerolog.TimeFieldFormat = time.RFC3339Nano
+	// zerolog.TimeFieldFormat = time.RFC3339Nano
```

Comment:
There doesn't seem to be another way to do this but this results in a data race
cc @abelanger5

Reply:
Can't we put this in an `init` method?

Reply:
yep works, good idea 👍